packages/www/pages/docs/guides/dashboard/ingest-playback-url-pair.mdx

```diff
@@ -0,0 +1,21 @@
+import DocsLayout from "../../../../components/DocsLayout";
+
+<DocsLayout>
+
+
+# How to pick an RTMP ingest and playback URL pair
+
+Your users or your application will need to use two Livepeer.com URLs, an RTMP ingest URL and a playback URL.
+
+1. The RTMP ingest URL is for the broadcaster to ingest the video stream, for example `rtmp://{region}-rtmp.livepeer.com/live/{stream-key}`.
+1. The playback URL is for the viewers to watch the stream, for example `https://{region}-cdn.livepeer.com/hls/{playbackId}/index.m3u8`.
+
+An RTMP ingest and playback URL pair references hardware in different parts of the world. This is important because the ingest point should be as physically close to the broadcaster as possible. Each region is three letters, the airport code of the server location. 
```

Comment:
Since we have MWD and FRA, might be important to add these as examples, and what regions they correspond to. Not everyone would understand the airport code reference. For example: "FRA stands for Frankfurt, and is our ingest point in the EU. Anyone streaming from the EU, Asia, the Middle East, and Africa would likely want to select this one. MDW stands for "Chicago Midway International Airport", an airport in Chicago. Anyone in the Americas would see better performance selecting this ingest point". 

Could be good to connect the dots for users more explicitly. Could include short examples such as .... "For example, if I am using Livepeer in Spain, I'd choose the Frankfurt ingest point. My ingest URL would then look like this:`rtmp://FRA-rtmp.livepeer.com/live/{stream-key}` and my playback URL would be: `https://FRA-cdn.livepeer.com/hls/{playbackId}/index.m3u8`. If I were streaming from Los Angeles, USA, I'd select URLs containing `MDW`."

Reply:
Good call. Updated that docs page with examples.